plrust/build

```diff
@@ -46,8 +46,8 @@ fi
         cd ./postgrestd
     fi
     rm --force rust-toolchain.toml
-    ./run clean
-    ./run install
+    STD_TARGETS="$STD_TARGETS" ./run clean
```

Comment:
I'm not sure why this acquired a `./run clean` given that `./run install` already does what should be enough but I am gonna figure that out later.